samples/sample-04/build.gradle

```diff
@@ -0,0 +1,85 @@
+buildscript {
+    ext {
+        kotlinVersion = "1.3.10"
+        springBootVersion = "2.1.1.RELEASE"
+        arrow_version = "0.8.1"
```

Comment:
What is this about?
Doesn't look like this property is used somewhere in the `buildscript` definition